bang/templates/items/costumeItem.html

```diff
@@ -0,0 +1,19 @@
+{% load tools %}
+{% load i18n %}
+
+{% with costume=item %}
+<a href="{{ costume.viewer_url }}" class="costume_link" data-ajax-url="{{ costume.ajax_viewer_url }}" data-ajax-title="{{ costume }}">
+    <div class="cos_image">
+        {% if costume.resolved_preview_image %}
+        <img alt="{{ costume }}" src="{{ costume.resolved_preview_image }}" height="200"/>
+        {% endif %}
+    </div>
+    <p class="cos_caption">
+        {% if costume.member %}
+        [{{ costume.t_name }}]<br>{{ costume.member.t_name }}
+        {% else %}
+        {{ costume.t_name }}
+        {% endif %}
```

Comment:
All this if/else can be replaces with `{{ costume }}`, which will call `__unicode__` and do exactly that.